src/main/kotlin/com/cognifide/gradle/aem/common/instance/InstanceActionPerformer.kt

```diff
@@ -1,29 +1,29 @@
 package com.cognifide.gradle.aem.common.instance
 
 import com.cognifide.gradle.aem.AemExtension
-import com.cognifide.gradle.aem.common.instance.action.AbstractAction
-import com.cognifide.gradle.aem.common.instance.action.AwaitAction
-import com.cognifide.gradle.aem.common.instance.action.ReloadAction
-import com.cognifide.gradle.aem.common.instance.action.ShutdownAction
+import com.cognifide.gradle.aem.common.instance.action.*
 
 /**
  * Executor for actions affecting multiple remote instances at once.
  */
 class InstanceActionPerformer(private val aem: AemExtension) {
 
-    fun await() = await {}
+    fun awaitUp(options: AwaitUpAction.() -> Unit = {}) = action(AwaitUpAction(aem), options)
 
-    fun await(options: AwaitAction.() -> Unit) = action(AwaitAction(aem), options)
+    fun awaitDown(options: AwaitDownAction.() -> Unit = {}) = action(AwaitDownAction(aem), options)
 
-    fun reload() = reload {}
+    fun reload(options: ReloadAction.() -> Unit = {}) = action(ReloadAction(aem), options)
 
-    fun reload(options: ReloadAction.() -> Unit) = action(ReloadAction(aem), options)
-
-    fun shutdown(options: ShutdownAction.() -> Unit) = action(ShutdownAction(aem), options)
-
-    fun shutdown() = shutdown {}
+    fun check(options: CheckAction.() -> Unit) = action(CheckAction(aem), options)
 
     private fun <T : AbstractAction> action(action: T, configurer: T.() -> Unit) {
         action.apply { notify = false }.apply(configurer).perform()
     }
+
+    // Aggregated / shorthands
```

Comment:
@szymon-owczarzak remember to use it when upgrading .. instead of just `reload()`